std/datetime.d

```diff
@@ -1075,6 +1075,25 @@ public:
         //assert(ist.opCmp(ist) == 0);
     }
 
+    /**
+     * Returns: A hash of the $(LREF SysTime)
+     */
+    size_t toHash() const @nogc pure nothrow @safe
+    {
+        if (_stdTime < 0)
+            return (cast(size_t) (_stdTime * -1)) + long.max;
```

Comment:
> Is it true for truncated time values, for example truncated to year?

I doubt that truncating to anything not base-2 related is going to make a huge difference.

> No. I mean only finalization step of MurmurHash2, not algorithms itself. It is very fast.

OK, I defer to you on this.